docs/pages/connect-your-client/tsh.mdx

```diff
@@ -124,24 +124,21 @@ To retrieve a user's certificate, execute:
 
 ```code
 # Full form:
-$ tsh login --proxy=proxy_host:<https_proxy_port>,<ssh_proxy_port>
+$ tsh login --proxy=proxy_host:<https_proxy_port>
```

Comment:
This is _supposed_ to work. See: https://github.com/gravitational/teleport/blob/d5e329e9ea144ebd4f3e1131da822b11c33d2132/lib/client/api.go#L1202

Unfortunately, `tsh`'s handling of custom ports is all kinds of broken. I tried to fix this months ago in #15581, but a bunch of tests broke.

I'm okay with removing this for now since a bug is preventing it from working, but if we do that we'll have to make sure we re-add this when the bug is fixed.

cc @jakule who helped me look into the tsh issues

Reply:
So should docs wait on this thread to see if we're gonna try to resolve the bug before processing this PR?

Reply:
@zmb3 I think we should merge this PR, so our docs are reflecting the current state. We should also revert it back when the `tsh` is fixed. Otherwise, people will keep reporting this bug to us as our docs say that it should work.